sql/telemetry_derived/events_live/view.sql

```diff
@@ -0,0 +1,39 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod.telemetry_derived.events_live` AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    document_id,
+    client_id,
+    normalized_channel,
+    normalized_country_code AS country,
+    environment.settings.locale AS locale,
+    normalized_app_name AS app_name,
+    metadata.uri.app_version AS app_version,
+    normalized_os AS os,
+    normalized_os_version AS os_version,
+    environment.experiments AS experiments,
+    sample_id,
+    payload.session_id AS session_id,
+    SAFE.TIMESTAMP_MILLIS(payload.process_start_timestamp) AS session_start_time,
+    payload.subsession_id AS subsession_id,
+    submission_timestamp AS `timestamp`,
```

Comment:
I think it would be consistent with what we have done for other tables to expose both `submission_timestamp` and `timestamp` in views, and prefer the use of `submission_timestamp`

Reply:
to be clear, this would mean exposing `submission_timestamp` both here and in the `telemetry.events_v1` view

Reply:
Okay – I'll file a bug for that and take care of it separately